www/contribute/how-tos/how-to-structure-and-style-large-poetic-productions.php

```diff
@@ -17,6 +17,20 @@
 					<li><p><a href="#a-single-collections-file">A single collections file</a></p></li>
 					<li><p><a href="#multiple-files">Multiple files</a></p></li>
 				</ol>
+				<li><p><a href="#modernizing-poetic-works">Modernizing poetic works</a></p></li>
+				<ol>
+					<li><p><a href="#modernize-prose-frontmatter-and-backmatter">Modernize prose frontmatter and backmatter</a></p></li>
+					<li><p><a href="#modernize-poems-carefully">Modernize poems <i>carefully</i>!</a></p></li>
+					<li><p><a href="#modernization-exceptions">Modernization exceptions</a></p></li>
+					<ol>
+						<li><p><a href="hyphenations-and-spacings">Hyphenations and spacings</a></p></li>
```

Comment:
I think all of these are missing `#` hashes?

Reply:
These have been fixed.